### ISSUE TITLE
Missing and inconsistent Data in public s3 for fim_3_0_34_1

### ISSUE THREAD

**Issue by mikejohnson51** [NOAA-OWP/inundation-mapping#687]
Hi FIM team,

This is directly related to #686. While that issue looked at the data holdings in `fim_4_0_7_0`, this issue relates to those in `fim_3_0_34_1`.

I am trying to pull down the FIM4 hydrologic network (catchments and flowpaths) for comparison with NextGen hydrofabric. I would expect to be able to find the catchment `POLYGON` and flowpath`LINESTRING` layers for each huc8 under the path pattern:

```
s3://noaa-nws-owp-fim/hand_fim/fim_3_0_34_1/outputs/fr/{huc8}/
``` 

###  Missing HUCs and variable dataset counts

Right now, 37 HUC8s are missing data completely. To find these I found the unique HUC8s represented in the NHDPlusV2 (first 8 digits of the REACHCODE) and listed their directories to count the available files (pattern below):

```
s3://noaa-nws-owp-fim/hand_fim/fim_3_0_34_1/outputs/fr/{huc8}/
``` 

When doing this, the following set had 0 data elements: 

```
 [1] "18020103" "18020106" "18020118" "18040005"
 [5] "14060001" "14060002" "04070001" "04060106"
 [9] "04060107" "04070002" "04090005" "04120104"
[13] "04150101" "04150303" "04150102" "04150309"
[17] "04150310" "04150302" "04150304" "04150305"
[21] "04150306" "04150307" "04150308" "04150401"
[25] "04150402" "04150403" "04150404" "04150405"
[29] "04150500" "04150406" "04150407" "04150408"
[33] "02010007" "04150409" "02060008" "08020201"
[37] "10150001"
```

It is also strange that the `{huc8}` directories have varying numbers of files ranging from 0 to 12:

```
table(net$fim_datasets)

#>  0   10   11   12 
#>  37   11 1248  821
```

## Expected behavior

I can find data for all HUC8s in CONUS and all HUC8s have consistent, complete, data holdings.

## Steps to replicate behavior (include URLs)

```r
pacman::p_load(glue, dplyr, aws.s3, nhdplusTools)

net = get_vaa('reachcode') %>%
  mutate(huc8 = substr(reachcode, 1,8)) %>% 
  select(huc8) %>% 
  distinct()

version = 'fim_3_0_34_1'

for(i in 1:nrow(net)){
  x = get_bucket_df(bucket = "noaa-nws-owp-fim", 
                    prefix = glue("hand_fim/{version}/outputs/fr/{net$huc8[i]}/"),
                    headers = list('x-amz-request-payer' = "requester")) 
  
  net$fim_datasets[i] = nrow(x)
  
  message(i, " of ", nrow(net))
  
}


table(net$fim_datasets)


filter(net, fim_datasets == 0)$huc8
```

**Comment by mikejohnson51** [comment]
Considering this closed with the movement towards #686 on my end. Please feel free to close if not useful to you all.

**Comment by RobHanna-NOAA** [comment]
Closed in favor of ticket 686 as mentioned above.
